docs/account/delete-account.mdx

```diff
@@ -28,7 +28,7 @@ If you have any questions regarding your data and the deletion of your account,
 
 
 <section class="explanation small-big">
-  <img src={useBaseUrl("img/wxm-devices/helium-bundle/delete-account.svg")} />
+  <img src={useBaseUrl("img/wxm-devices/helium-bundle/delete-account.jpg")} />
```

Comment:
@epapangelis Please delete the old .svg file if it's not needed anymore in any other occasion.